Sources/Logging/Logging.swift

```diff
@@ -392,8 +400,21 @@ extension Logger {
 }
 
 extension Logger.Level: Comparable {
+    private var severityOrdering: Int {
+        switch self {
+        case .debug: return 0
```

Comment:
So I was very worried about the code-size that this will generate but the Swift compiler is really smart and as long as the order of the cases is aligned with the numbers, this works really well as the compiler can see through this.

Reply:
https://twitter.com/migueldeicaza/status/1109173266483036160